pkgs/development/python-modules/jupyterlab/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, isPy3k, fetchPypi, ipython_genutils, jupyterlab_launcher, notebook }:
+buildPythonPackage rec {
+  pname = "jupyterlab";
+  version = "0.31.12";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hp6p9bsr863glildgs2iy1a4l99m7rxj2sy9fmkxp5zhyhqvsrz";
+  };
+
+  propagatedBuildInputs = [
+    ipython_genutils
+    jupyterlab_launcher
+    notebook
+  ];
+
+  # No tests in archive
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Jupyter lab environment notebook server extension.";
+    license = with licenses; [ bsd3 ];
+    homepage = "http://jupyter.org/";
```

Comment:
no quotes

Reply:
There is no consensus on that in nixpkgs. URL literals get transformed into strings so it doesn't make any practical difference. This is unlike path literals which do have a different type.

Reply:
There is a strong opinion to get rid of the type and use strings instead, but no such (formal) decision has been made. Would it not make sense to keep using the dedicated type for it until decided otherwise? I mean, if people want this they can open an RFC instead of increasing incoherence in the package set.

Reply:
I agree that we should have the discussion at some point.

nixpkgs is already incoherent so going one way or another doesn't make much difference for new packages. In either case we can write a script that converts existing packages once the above discussion has been resolved.